packages/blockchain/src/index.ts

```diff
@@ -334,9 +334,8 @@ export default class Blockchain implements BlockchainInterface {
     }
 
     if (!genesisBlock) {
-      const common = this._common.copy()
-      common.setHardforkByBlockNumber(0)
-      genesisBlock = Block.genesis({}, { common })
+      this._common.setHardforkByBlockNumber(0)
+      genesisBlock = Block.genesis({}, { common: this._common })
```

Comment:
I was already sceptical during the review when seeing this direct `Common` manipulation (I already spotted this in some former PR and assumed this to be a mistake and changed this), but during the review I thought: if he is explicitly going back here he will have some stronger reason for it.

But this is actually why the VM API tests fail, switched this back to use the copied version for testing and then the tests pass.

Is there a specific reason you want this to manipulate the initial Common? 🤔

Reply:
hmmm yes the change was motivated because without it I was getting an issue putting and getting the genesis block in initializing the `taunus` chain because otherwise the default hardfork was on `istanbul` and not allowing `baseFeePerGas` to be set (as defined in the `taunus` geth genesis params)

This change allowed the blockchain to be on the right hardfork, without it it fails with:

```shell
➜  client git:(merge-contd) npm run client:start -- --gethGenesis=./test/testdata/geth_genesis_merge_interop_taunas.json

> @ethereumjs/client@0.1.0 client:start
> ts-node bin/cli.ts "--gethGenesis=./test/testdata/geth_genesis_merge_interop_taunas.json"

INFO [10-02|00:38:14] Data directory: /Users/rg/Library/Ethereum/ethereumjs
INFO [10-02|00:38:14] Initializing Ethereumjs client...
INFO [10-02|00:38:14] Full sync mode
INFO [10-02|00:38:14] Connecting to network: geth_genesis_merge_interop_taunas
ERROR [10-02|00:38:14] Error: A base fee for a block can only be set with EIP1559 being activated
    at new BlockHeader (/Users/rg/dev/ethereumjs-monorepo/packages/block/src/header.ts:252:15)
    at Function.fromValuesArray (/Users/rg/dev/ethereumjs-monorepo/packages/block/src/header.ts:169:12)
    at Function.fromRLPSerializedHeader (/Users/rg/dev/ethereumjs-monorepo/packages/block/src/header.ts:133:24)
    at DBManager.getHeader (/Users/rg/dev/ethereumjs-monorepo/packages/blockchain/src/db/manager.ts:191:24)
```

I'll take a look at the VM errors though and see if I can make them work together, maybe in the morning I can think of a suitable alternative.